services/motion/builtin/move_request.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	defaultReplanCostFactor = 1.0
+	defaultReplanCostFactor = 2.0
```

Comment:
This was necessary to get MoveOnGlobe to work at all during outdoor hardware testing. Do we want to make this change here or in a different PR? Does this require a ticket?